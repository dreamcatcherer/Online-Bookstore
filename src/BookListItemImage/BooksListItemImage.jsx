import React from "react";
import css from "./BooksListItemImage.module.css"

const BooksListItemImage = ({url, title}) => {
  return (
    <div className = {css.imageStyle}>
      <img src={url} alt={title} />
    </div>
  );
};

export default BooksListItemImage;
