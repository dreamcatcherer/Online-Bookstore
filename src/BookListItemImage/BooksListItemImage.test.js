import React from "react";
import { render } from "@testing-library/react";
import BooksListItemImage from "./BooksListItemImage.jsx";

it("renders the BooksListItemImage component correctly", () => {
  const { container } = render(
    <BooksListItemImage
      url="https://images-na.ssl-images-amazon.com/images/I/9195S4FTnOL.jpg"
      title="Potty"
    />
  );
  expect(container).toMatchSnapshot();
});
