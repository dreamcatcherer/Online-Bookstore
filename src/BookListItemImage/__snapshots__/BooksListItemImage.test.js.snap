// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the BooksListItemImage component correctly 1`] = `
<div>
  <div
    class="imageStyle"
  >
    <img
      alt="Potty"
      src="https://images-na.ssl-images-amazon.com/images/I/9195S4FTnOL.jpg"
    />
  </div>
</div>
`;
